frontend/src/components/form/BaseInput.vue

```diff
@@ -11,6 +11,8 @@ defineEmits(["update:modelValue"]);
 let dd = 7;
 
 type inputType = "text" | "password" | "email" | "number" | "date";
```

Comment:
The destructuring assignment is used to extract these props into their own variables so they can be used directly in the component. you can use `modelValue` directly , instead of having to use `props.modelValue`.